cmd/buildkitd/debug.go

```diff
@@ -28,6 +29,8 @@ func setupDebugHandlers(addr string) error {
 		bklog.G(req.Context()).Debugf("triggered GC from debug endpoint")
 	}))
 
+	m.Handle("/metrics", promhttp.Handler())
```

Comment:
I was wondering myself why not `/debug/metrics` as route but this is part of the debug handler so good to me.

Reply:
It's mostly because `/metrics` is the typical endpoint for prometheus. I don't think it matters though so I can always move it to `/debug/metrics`.

The main reason why it's in the debug handler is because buildkit doesn't expose an HTTP server otherwise. At the same time, this pathway is only relevant for buildkit when it's run in isolation. If buildkit is embedded, such as moby, we'll piggyback on their own `/metrics` endpoint and be included there.